package.json

```diff
@@ -5,39 +5,51 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build --configuration=production --project=cps-ui-kit",
+    "build:documentation": "ng build --configuration=production --project=composition --base-href '/cps-shared-ui/'",
     "watch": "ng build cps-ui-kit --watch --configuration development",
     "test": "npm run cypress:run",
     "lint": "eslint \"**/*.ts\"",
     "cypress:open": "cypress open --component --browser chrome",
-    "cypress:run": "cypress run --component"
+    "cypress:run": "cypress run --component",
```

Comment:
these are not needed I assume:
```
"dev:ssr": "ng run composition:serve-ssr",
 "serve:ssr": "node dist/composition/server/main.js",
 "build:ssr": "ng build && ng run composition:server",
 "prerender": "ng run composition:prerender"
```

Reply:
it's preparation for using prerendering is it necessary to remove it?

Reply:
Ok, let's keep it